### PR TITLE
Changed file to to fix bug where fdp base uri could not be replaced

### PR DESCRIPTION
**Pull request by Shamanou**
- Only small changes where needed, removed return and moved getfaircalaogsmethod downwards

### CODE REVIEW

File: extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js [hunk 1]
```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 			self._distributionDiv.html('');
 			self._pushtoResourceDiv.html('');
 			self.uri = self._baseUriSpan.text();
-			getFairCatalogs(self._baseUriSpan.text(), self);
-			return;
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js:129]: Removing the `return` statement might have some side-effects?

File: extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js [hunk 2]
```diff
@@ -125,11 +125,10 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
...
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
+		getFairCatalogs(self._baseUriSpan.text(), self);
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js]: Could you explain a bit more about this fix? When was the if clause above called? As far as I can see, the baseuri length > 7 conditional is true even if you add a new url?

**Review by kburger — COMMENTED** [review]
Some questions, the cause is not entirely clear to me.

File: extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js (hunk 1 above)
Comment (Shamanou) [reply extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js:129]: i do not foresee unexpected side effects which ones are you thinking off?

File: extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js (hunk 2 above)
Comment (Shamanou) [reply extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js]: the clause is called when the user presses the apply button within the push to fdp dialog, the user first presses the edit button to change the baseuri. The `self._replaceBaseUri` call takes care off setting the new `self.fairDataPointPost.baseUri`

**Comment by Shamanou** [comment]
The cause whas the `return;` within the if-block

File: extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js [hunk 3]
```diff
@@ -128,7 +128,6 @@ fairDataPointPostDialog.prototype._editBaseUri = function(src) {
 		}
 		MenuSystem.dismissAll();
 		self._replaceBaseUri(newBaseUri, false);
-		getFairCatalogs(self._baseUriSpan.text(), self);
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/fairDatapointPost.js]: Was this line supposed to be removed? Seems like the retrieval of the catalogs is not triggered from the apply event now.

**Review by kburger — APPROVED** [review]

**Pull request merged**